CorsixTH/CMakeLists.txt

```diff
@@ -279,5 +279,10 @@ if(NOT USE_SOURCE_DATADIRS)
       SET(BU_CHMOD_BUNDLE_ITEMS ON)
       FIXUP_BUNDLE(${CMAKE_INSTALL_PREFIX}/CorsixTH.app \"\" \"\")
       ")
+    if(WITH_LUAROCKS)
+      install(CODE "execute_process(
+        COMMAND bash -c ${CMAKE_SOURCE_DIR}/scripts/macos_luarocks ${CMAKE_INSTALL_PREFIX})
```

Comment:
In order for bash to handle spaces in the file path, this would be better as
`COMMAND bash \"${CMAKE_SOURCE_DIR}/scripts/macos_luarocks\" \"${CMAKE_INSTALL_PREFIX}\")`